src/epoch.cpp

```diff
@@ -190,42 +247,22 @@ std::ostream &operator<<(std::ostream &s, const epoch &ep)
     return s;
 }
 
-bool operator>(const epoch &c1, const epoch &c2)
-{
-    return c1.m_tp > c2.m_tp;
-}
-bool operator<(const epoch &c1, const epoch &c2)
-{
-    return c1.m_tp < c2.m_tp;
-}
-bool operator>=(const epoch &c1, const epoch &c2)
-{
-    return c1.m_tp >= c2.m_tp;
-}
-bool operator<=(const epoch &c1, const epoch &c2)
-{
-    return c1.m_tp <= c2.m_tp;
-}
-bool operator==(const epoch &c1, const epoch &c2)
-{
-    return c1.m_tp == c2.m_tp;
-}
-bool operator!=(const epoch &c1, const epoch &c2)
-{
-    return c1.m_tp != c2.m_tp;
-}
-
-kep_clock::duration operator-(const epoch &lhs, const epoch &rhs)
+microseconds operator-(const epoch &lhs, const epoch &rhs)
 {
     return lhs.m_tp - rhs.m_tp;
 }
 
-kep_clock::time_point epoch::get_tp() const
+time_point epoch::get_tp() const
 {
     return m_tp;
 }
-epoch utc_now() {
-    return epoch(kep_clock::utc_now());
+epoch utc_now()
```

Comment:
so we leave utc in the name at the end? I think is better, since the way the clock works under the hood is irrelevant, the timepoint returned is the UTC time.

Reply:
No, it was not my intention to leave "utc" in as there is nothing specific to UTC in the returned time point.

I would like to make this a static method of ``epoch``, so that it would be called as ``epoch::now()``. Is that ok?

Reply:
also fine. Go ahead.